.github/workflows/e2e.yml

```diff
@@ -661,14 +654,14 @@ jobs:
       e2e-leg-8-vcluster-online,
       e2e-leg-9-vcluster,
       e2e-leg-10-vcluster,
-      e2e-server-upgrade-admintools,
       e2e-server-upgrade-vcluster,
       e2e-operator-upgrade,
-      e2e-udx-admintools,
       e2e-udx-vcluster
     ]
     runs-on: ubuntu-latest
     if: ${{ always() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && contains(github.ref, 'k8s-sync') && (github.event_name == 'push')}} 
+    concurrency:
```

Comment:
Have you tested this one?

Reply:
Remove it for now and will put the sync-vcluster job into a queue in the future.